pkg/agent/flowexporter/flowrecords/flow_records.go

```diff
@@ -35,9 +35,32 @@ func NewFlowRecords(connStore *connections.ConnectionStore) *FlowRecords {
 }
 
 // BuildFlowRecords builds the flow record map from connection map in connection store
-func (fr *FlowRecords) BuildFlowRecords() error {
+func (fr *FlowRecords) BuildFlowRecords(v4Enabled bool, v6Enabled bool) error {
```

Comment:
We do not need the args. From IP in key, we can say if it is v4 or v6.

Reply:
Updated.